ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Chip from '@material-ui/core/Chip'
+import LOSSLESS_FORMATS from './consts'
+
+export const QualityInfo = (props) => {
```

Comment:
You haven't done yet the deconstructing here, to pass `rest`  to the Chip component. If you have questions about this, ping me on Discord.